nwb_conversion_tools/datainterfaces/behavior/movie/moviedatainterface.py

```diff
@@ -152,49 +142,55 @@ def run_conversion(
             f"({len(image_series_kwargs_list)}) vs. file_paths ({len(self.source_data['file_paths'])})!"
         )
 
-        for j, file in enumerate(file_paths):
-            timestamps = starting_times[j] + get_movie_timestamps(movie_file=file)
-
-            if len(starting_times) != len(file_paths):
-                starting_times.append(timestamps[-1])
+        # check for duplicates in image_series_kwargs list keys:
+        def _check_duplicates(image_series_kwargs_list):
+            image_series_kwargs_list_keys = [i["name"] for i in image_series_kwargs_list]
+            if len(set(image_series_kwargs_list_keys)) < len(image_series_kwargs_list_keys):
+                assert external_mode, "for multiple video files under the same ImageSeries name, use exernal_mode=True"
+            keys_set = []
+            image_series_kwargs_list_unique = []
+            for no, image_series_kwargs in enumerate(image_series_kwargs_list):
```

Comment:
```suggestion
            for n, image_series_kwargs in enumerate(image_series_kwargs_list):
```
I'd avoid using `no` since it's a standalone English word with different contextual meaning related to boolean archtypes.

Some common iteration variable names, especially for `enumerate`, are `i` (though more common in C/C++/C#), `j`, `k`, `n`, `m`. Basically anything that is also a common integer indexing variable from discrete & combinatorial mathematics.